### ISSUE TITLE
Refreshing maintainer list

### ISSUE THREAD

**Issue by jakirkham** [conda-forge/ucx-split-feedstock#190]
Over time various folks have been involved in maintenance here. As is typical in open source, some folks have moved on (and hope things are going well for them!). However this does mean the maintainer list can get a bit stale. So would like to refresh it with folks who would like to be active here (and free those who would like to drop 🙂). Here is the current list

https://github.com/conda-forge/ucx-split-feedstock/blob/b60fb426784cbb6d5cbd384a76c86920737838f0/recipe/meta.yaml#L94-L102

<hr>

To facilitate updating the maintainer list, if you would like to remain a maintainer, could you please check the box below next to your username? Also if you have been active here and would like to be a maintainer, please comment below and we can add you to the list

- [ ] @marcinz 
- [ ] @Akshay-Venkatesh 
- [ ] @jakirkham 
- [ ] @quasiben 
- [ ] @TomAugspurger 
- [ ] @leofang 
- [ ] @matthiasdiener 
- [ ] @xhochy

As folks get busy, will give this 2 weeks from the date this issue is opened. At that point will make updates based on the current list

If for whatever reason folks were not able to respond in that 2 week window, we can always do follow up changes after

Thanks all for your help! 🙏

**Comment by xhochy** [comment]
I feel like we're touching some critical here. Normally, maintainers are not removed except by themselves. For now, I felt that maintainership in conda-forge was given for life, not limited to a certain duration/level of activity.

**Comment by jakirkham** [comment]
This is modeled after the same thing done with `conda` maintainers: https://github.com/conda-forge/conda-feedstock/issues/188

To be fair the goal here is not to push people out. Would be happy to learn everyone on this list plans to be actively involved. The reality is some folks have moved on. We wish them the best and thank them for the work done to get us to this state. So this is more to free their inboxes and let them off the hook. I would rather this not become a burden to them

**Comment by pentschev** [comment]
Feel free to add me to the maintainer's list as well.

**Comment by xhochy** [comment]
> Maybe we can add a new `extra:emeritus-mantainers` section and make conda-smithy render this list in README. Then we can ask existing maintainers if they want to retire. This does not require any change in conda or conda-build.

Personally, I would love this, but we should make this a CFEP (enhancement proposal) and not do it on an individual basis. I would like to move to emeritus in quite some feedstocks.

**Comment by leofang** [comment]
Indeed, while it'd be a conda-forge only decision, the buy-in requires some changes in the tool (conda-smithy at least), which may require a process like CFEP.
